chart/kubeapps/templates/dashboard-config.yaml

```diff
@@ -23,4 +23,5 @@ data:
   config.json: |-
     {
       "namespace": "{{ .Release.Namespace }}"
+      "kubeappsVersion": "{{ .Chart.AppVersion }}"
```

Comment:
maybe calling this `appVersion` would be enough.

Reply:
You might also need to update the development version of this file.
```
src/public/config.json
```

Reply:
what's that file for?

Reply:
If I remember correctly, this was a file that is required if you swap the dashboard with telepresence or something like that. @prydonius might have more info about it.